frappe/desk/doctype/auto_repeat/auto_repeat.js

```diff
@@ -72,4 +80,17 @@ frappe.ui.form.on('Auto Repeat', {
 			}
 		});
 	}
-});
\ No newline at end of file
+});
+
+function render_schedule(frm){
```

Comment:
not a good idea to have non namespaced functions like these

either add it to the frm object or maybe

```
frappe.provide("frappe.auto_repeat");
frappe.auto_repeat.render_schedule = function(frm) { .. }
```